### DISCUSSION TITLE
Contributor Call: 18 February 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3516] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 18 February 2023 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/feb-18-2023/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
TBD

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark 
- @lexander2002 
- @joostdecock 
- @KaerMorhan 
- @bobgeorgethe3rd 
- @BenJamesBen 
- @darigovresearch 
- @nataliasayang 
- @nchilada 
- Lizzomatic
- ArmaUrschweiz

✨ **Lex**
I've deleted the "under maintenance" discord channels (they were only visible to admins, or contributors - I'm not sure - so don't mind if this doesn't ring a bell) because they have been "under maintenance" for quite some time.

✨ **Lizzo**
I’ve been learning React for a couple of weeks and I was wondering if somebody is interested in pairing up so I can look at the process.
_Joost: Yes, you can send me a message, we will figure out the time. If other people are interested, we can make it a group thing and not a one-on-one thing._
_Lizzo: I’ve been interested in how the patterns are build._
_Joost: Pattern are not built in React, we use JavaScript and render it through React. So if you want to learn React, it’s a nice idea to learn by watching me but if you just want to learn it for FS, you don’t have to worry about it that much. UI elements are built in React._
_Karen: I’m interested in what patterns would you like to add._
_Lizzo: One of them is a vest and another one is a cardigan with pleats at the shoulder._

✨ **Karen**
There is a newsletter coming in a month-month and a half. Benjamin, would you like to be interviewed?
_Ben: Yes. I saw the message but didn’t respond to it yet._

Is there something people would like to see on the newsletter on April 1st?
_Bob: Maybe a section about pattern that are in progress? Mentioning Zander, for example._
_Karen: I know Zee has a Jane pattern that will be launched with V3 also._
_Joost: We have talked a lot about V3, maybe we’ll have something to show until April 1st._

I tried to catch up on showcases, they’re waiting in Strapi, maybe we can highlight some of them.
_Joost: We can deploy them on the website, we don’t have to wait until V3. I abandon those things sometimes but I also have a job and all of that._
_Karen: You’re not a bottleneck for anything. We could use showcases for the newsletter, they have some cool pictures._
_Bob: Maybe you could do highlighting for Discord._
_Ben: We could post a list of 20 or so candidate "Best of Discord" threads to Discord and have people give input on which ones to include in the newsletter._

✨ **Arma**
_Karen: We have some new people, would you like to introduce yourself?_
Hello, I was a patron but then I stopped because of Discord, and now I’m back. I’m a sewist and a translator.
_Karen: Welcome back!_
_Joost: If you want, I can update you briefly on what was happening._

As I wasn’t active for two years, where is the translting done now? And also what is the main goal at the moment?
_Joost: We’re changing coding for translations but from the trnaslators’ point of view nothing will change. We’re doing it on Crowdin. Content translation is the same, but the coding with strings will change as we don’t know if those strings are used. So we’re bringing the translation closer to the source material so we can drop strings that are not used. It becomes easier on the maintenance side.
We’re working on V3 of FS. Why V3? FS practices semantic versioning, so we have major, minor and patch versions. A lot of ideas accumulated on V2 that we started working on V3. The core library is mostly ready, now we’re working on the frontend, website. We’re also writing a new backend which simplifies working with it, we’re using SQLite now. SQlite is an open source but you can’t contribute. Because of this FS can have local versions that don’t require Internet connection. Our new website will have all the languages on one website. V3 sneak peek: https://next.freesewing.org/signup_
_Arma: Are the patterns kept on hold until V3?_
_Joost: It’s not a deliberate limitation, it’s more that it doesn’t make sense to build a pattern on V2 which will be discarded for the most part but V3 is not ready enough to build a pattern on it._

✨ **Bob**
Option groups are now more visible. Do we want to order them in a specific way or will we keep the alphabetical order?
_Joost: For now, I’m not interested in focusing on the icing for the cake because I’m still baking the cake. Alphabetical seems logical for me but also it will need to be alphabetical in translation which is hard but we’ll manage that._

✨ **Nikhil**
I’m working on a side project that might answer fit questions from people.

✨ **Ben** created an issue #3553

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
